.github/actions/ghcr-docker-login/action.yml

```diff
@@ -14,7 +14,47 @@ outputs:
 runs:
   using: "composite"
   steps:
-    - name: Login to GitHub Container Registry
+    - name: Login to GitHub Container Registry (attempt 1)
+      id: login1
+      continue-on-error: true
+      uses: docker/login-action@5e57cd118135c172c3672efd75eb46360885c0ef # v3.6.0
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ inputs.github_token }}
+
+    - name: Login to GitHub Container Registry (attempt 2)
```

Comment:
Should we not sleep in-between here?

Reply:
We've probably already waited `TIMEOUT` seconds, which I believe is... 30? I don't know if it makes sense to wait longer after.